django_rq/urls.py

```diff
@@ -1,21 +1,24 @@
-from django.conf.urls import patterns, url
```

Comment:
Could you please revert this file back to how it was? I prefer the old approach as it has less imports :)